### PR TITLE
doc: add minutes for meeting 23 June

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

**Review by Trott — APPROVED** [review]
LGTM with recording link added (which will resolve the lint issue)

**Review by gireeshpunathil — APPROVED** [review]

File: meetings/2021-06-23.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# Node.js Technical Steering Committee (TSC) Meeting 2021-06-23
+
+## Links
+
+* **Recording**:
```
Comment (bl-ue) [inline meetings/2021-06-23.md:5]:
```suggestion
* **Recording**:
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

**Comment by mhdawson** [comment]
@Trott who's uploading the video as I don't see it there yet.

**Comment by Trott** [comment]
> @Trott who's uploading the video as I don't see it there yet.

/ping @mmarchini

**Review by tniessen — APPROVED** [review]

**Comment by mhdawson** [comment]
@mmarchini are was there a recording that can be uploaded? If not we'll just land this as is.

**Comment by mhdawson** [comment]
I checked today and I don't see a video on youtube. I'm going to land as is.

**Pull request merged**